Lundalogik/lime-elements
# fix(menu): allow clicks for permitted elements from open menu surface

When the click event originates from a component that is allowed, such as a chip-set, then the menu should allow the click. It should only kill click events that originate from elements that are not allowed.

Fixes [Table filters bugs out immediately](https://github.com/Lundalogik/crm-feature/issues/3678)

How to test:
You should be able to remove chips now even if the menu is open. 

https://github.com/Lundalogik/lime-elements/assets/122509723/3675a66b-e9c4-40e7-91e2-f25ac6819d5e

Also manually test that other components using the menu-surface still work as intended. I've manually tested but it can't hurt to test a bit more.
